pkg/store/state.go

```diff
@@ -0,0 +1,70 @@
+package store
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+)
+
+type OIDCState struct {
+	Status       int    `json:"status"`
+	AccessToken  string `json:"access-token"`
+	IDToken      string `json:"id-token"`
+	RefreshToken string `json:"refresh-token"`
+
+	RequestID   string `json:"req-id"`
+	RequestPath string `json:"req-path"`
+	OAuthState  string `json:"oauth-state"`
+	Scheme      string `json:"scheme"`
+}
+
+// Iota token
+const (
+	StatusNeedToken = iota
+	StatusTokenReady
+)
+
+// NewState .... create new state to store token for OIDC
```

Comment:
Can you please update these comments to be of the following form?
```go
// XFunc does the thing that it does.
```

I agree that a bunch of them are pretty simple, but it doesn't hurt to explain what they do and are used for regardless.